core/src/avm2/globals/XMLList.as

```diff
@@ -21,11 +21,24 @@ package {
         AS3 native function text():XMLList;
         AS3 native function toXMLString():String;
         AS3 native function toString():String;
+        AS3 native function comments():XMLList;
+        AS3 native function processingInstructions(name:String = "*"):XMLList;
 
         // The following native methods are not declared in the documentation,
         // but still exist
         AS3 native function name(): Object;
 
+        AS3 function toJSON(k:String) : *
+        {
```

Comment:
nit: The rest of the file puts the bracket on the same line as the function definition, like `AS3 function toJSON(k:String):* {`

Reply:
Fixed